src/main/java/org/takes/rq/RqWithDefaultHeader.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
```

Comment:
@dmzaytsev hmm, I think we agreed that it would be centraly changed everywhere - so here we need it to have untouched - i.e. 2015. Am I wrong?